app/js/components/LabOrdersList.jsx

```diff
@@ -154,6 +174,34 @@ export class LabOrdersList extends PureComponent {
     dispatch(filtersAction.setLabOrdersListFilters(newFilters));
   }
 
+  async handleCancel(order) {
+    const {
+      currentProvider,
+      dispatch,
+    } = this.props;
+    const cancelConfirmation = await swal("Are you sure you would like to cancel this order ?", {
+      buttons: {
+        YES: "YES",
+        NO: 'NO',
+      },
+    });
+    if (cancelConfirmation === "YES") {
+      const cancelledOrder = {
+        careSetting: order.careSetting,
+        concept: order.concept,
+        patient: order.patient.uuid,
+        encounter: order.encounter,
```

Comment:
Does it still work if you remove this "encounter" line as well?  If so, let's remove it... technically the discontinue order is not part of the same encounter has the original order, so it should be fine for the encounter to be null.

Reply:
@mogoodrich no it doesn't, it throws an invalid submission error, needing an encounter